flood_api/utils/validation_helpers.py

```diff
@@ -43,7 +83,43 @@ def validate_dates(start_date: date, end_date: date) -> None:
     Returns:
     None
     """
-    date_range_valid = start_date <= end_date
+    date_range_is_valid = start_date <= end_date
 
-    if not date_range_valid:
-        raise HTTPException(status_code=400, detail="Invalid date range")
+    if not date_range_is_valid:
+        raise HTTPException(
+            status_code=INVALID_STATUS_CODE, detail="Invalid date range"
+        )
+
+
+def validate_generic_inputs(*args: tuple, possible_inputs_str: str) -> None:
+    """
+    Check if exaclty one of the given inputs is valid, i.e. not None.
+    If not, raise an HTTPException with status code 404.
```

Comment:
I think a 400 status code may be more apropriate in this case since it is a problem with the request rather than missing data.

Reply:
True... 404 was used because it had been previously suggested in a similar (but different) [context](https://github.com/openearthplatforminitiative/flood-api/pull/5#discussion_r1387701475). 400 is probably better in this case.

Reply:
Changed to 400